serverless/pages/get-started-general.asciidoc

```diff
@@ -0,0 +1,36 @@
+[[serverless-get-started]]
+= Get started
+
+There are two options to create serverless projects:
+
+* If you are an existing customer, https://cloud.elastic.co/login[log in to Elastic Cloud]. On the home page, you will see a new option to create serverless projects. Note that if you are already subscribed to Elastic Cloud, there is no specific trial for serverless projects.
```

Comment:
So existing customers start with a trial or they simply are already signed up and can just use serverless (no trial). That is, if I create my first project do I get 14 days free?

Reply:
removed the left over note to clear up confusion.